contrib/prometheus/prometheus-all.json

```diff
@@ -62,6 +59,23 @@
                             {
                                 "name": "KUBERNETES_RO_SERVICE_PORT",
                                 "value": "8001"
+                            },
+                            {
+                                "name": "PUSHGATEWAY_SERVICE_HOST",
+                                "value": "localhost"
+                            },
+                            {
+                                "name": "PUSHGATEWAY_SERVICE_PORT",
+                                "value": "9091"
+                            }
+                        ],
+                        "image": "jayunit100/kube-prometheus",
```

Comment:
side note: Should we talk with prom upstream to mock their docker files for kube?